src/k8s/04_csi_key_vault_access_policy.tf

```diff
@@ -0,0 +1,7 @@
+resource "azurerm_key_vault_access_policy" "cluster_access_policy" {
+  key_vault_id = data.azurerm_key_vault.kv.id
+  tenant_id    = data.azurerm_subscription.current.tenant_id
+  object_id    = "25859118-4d19-418a-97ff-97af3a4ca929"
```

Comment:
can we make this value general? if it's complex get managed identity we can create a custom user assigned managed identity